components/Workspaces/TrackedRepoWizard/SelectOrgReposStep.tsx

```diff
@@ -55,7 +55,9 @@ export const SelectOrgReposStep = ({
   const [filteredRepositories, setFilteredRepositories] = useState<Map<string, boolean>>(repositories);
 
   useEffect(() => {
-    setFilteredRepositories(new Map([...repositories.entries()].filter(([repo]) => repo.startsWith(organization))));
+    setFilteredRepositories(
+      new Map([...repositories.entries()].filter(([repo]) => repo.toLowerCase().startsWith(organization)))
```

Comment:
Needed to filter lowercased. Prior to this fix, the orgs users were a part of had the correct casing so this wasn't necessary, but now that we allow for user inputted orgs to search for, this is required.